hooks/pre/29-el7_upgrade_postgresql.rb

```diff
@@ -0,0 +1,4 @@
+# On EL7 the SCL override should be used instead
+if !app_value(:noop) && el7
+  FileUtils.rm_f('/etc/systemd/system/postgresql.service.d/postgresql.conf')
```

Comment:
Should we also `FileUtils.rmdir('/etc/systemd/system/postgresql.service.d/')`? Probably with a `rescue`, as it might be non-empty, but then it's not our fault. (The rm of the conf file is sufficient for fixing the bug, this one is just cosmetics)

Reply:
I thought about that, but given EL7 is on its way out I was a bit lazy.